services/prometheus/scrape_configs.go

```diff
@@ -94,6 +94,21 @@ func scrapeConfigForPMMManaged(interval time.Duration) *config.ScrapeConfig {
 	}
 }
 
+func scrapeConfigForQANAPI2(interval time.Duration) *config.ScrapeConfig {
+	return &config.ScrapeConfig{
+		JobName:        "qan-api2",
+		ScrapeInterval: model.Duration(interval),
+		ScrapeTimeout:  scrapeTimeout(interval),
+		MetricsPath:    "/debug/metrics",
+		ServiceDiscoveryConfig: sd_config.ServiceDiscoveryConfig{
+			StaticConfigs: []*targetgroup.Group{{
+				Targets: []model.LabelSet{{addressLabel: "127.0.0.1:9933"}},
+				Labels:  model.LabelSet{"instance": "qan-api2"},
```

Comment:
That should be `pmm-server` too. Job name is different, that's enough for Prometheus.

Reply:
Fixed.